tests/BasicSearchTests.FunctionalTests.Core/EnvironmentSettings.cs

```diff
@@ -54,7 +54,7 @@ public static string ConfigurationName
             {
                 if (string.IsNullOrEmpty(_configurationName))
                 {
-                    _configurationName = GetEnvironmentVariable("ConfigurationName", required: true);
+                    _configurationName = GetEnvironmentVariable("ConfigurationName", defaultValue: "Dev");
```

Comment:
This is a convenience change. If you didn't open Visual Studio with the environment `ConfigurationName` set, your tests would fail. Now it'll default to the `Dev.json` file.